### PR TITLE
Code completion functional

### PR DESCRIPTION
**Pull request by Georgehe4**
screen shot 2015-07-13 at 1 30 17 pm

@devoncarew @lukechurch

### CODE REVIEW

**Comment by Georgehe4** [comment]
#570

File: lib/embed/embed.dart [hunk 1]
```diff
@@ -54,6 +56,10 @@ class PlaygroundMobile {
   Gist backupGist;
   Router _router;
 
+  Document _dartDoc;
```
Comment (devoncarew) [inline lib/embed/embed.dart]: Are these three fields referenced? I can't see where they're used in the source.

**Comment by devoncarew** [comment]
lgtm; one question about potentially unused vars.

**Comment by Georgehe4** [comment]
#575 Bug that came up while testing
Merging.

**Pull request merged**